c++/blocks.rst

```diff
@@ -165,6 +165,21 @@ Below the header there are some tabs showing:
 
         .. image:: /_static/img/c++/dependency_graph.png
 
+Biicode badge
+^^^^^^^^^^^^^
+
+Something useful for the spread of your blocks is the badge.
```

Comment:
A biicode status badge is a dynamically generated image displaying your block's latest published version in biicode.